synapse/lib/stormlib/project.py

```diff
@@ -293,6 +360,18 @@ async def iter(self):
 
 @s_stormtypes.registry.registerType
 class ProjectSprint(s_stormtypes.Prim):
+    '''
+    Implements the Storm API for a ProjectSprint
+    '''
+
+    _storm_locals = (
+        {'name': 'name', 'desc': 'The name of the sprint',
+         'type': 'str'},
+        {'name': 'desc', 'desc': 'A description of the sprint',
+         'type': 'str'},
+        {'name': 'tickets', 'desc': 'Yields out the tickets associated with the given sprint',
+         'type': 'Yields'},
```

Comment:
So this isn't a function that yields, but it's not a list. Is there a convention in the stormdocs for this?

Reply:
`lib.lift.byNodeData` is similar, might be able to do something like the docs for that?

Reply:
Sorta? `byNodeData` is also meant to be used as a function, while `tickets` we treat as a property that yields things out of, no function call necessary. 

It's a slight difference. But one worth noting.

Reply:
Still a bit different since that's a function but I'm not sure there's anything closer.

Reply:
There isn't, but also in just trying that, it screams on the assertion error we have in there.

So I think for now, I'll tweak the type to a generator just to be specific, and leave it at that.

Reply:
currently stor / ctors are underdocd and we don't have an existing convention for them.